ydb/core/ydb_convert/ydb_convert.h

```diff
@@ -64,4 +65,20 @@ bool FillACL(NKikimrSchemeOp::TModifyScheme& out,
 void FillOwner(NKikimrScheme::TEvModifySchemeTransaction& out,
     const TMaybeFail<Ydb::Scheme::ModifyPermissionsRequest>& in);
 
+struct TConversionTypeInfo {
+    NScheme::TTypeInfo TypeInfo;
+    TString TypeMod;
+    bool IsNotNull;
+};
+
+bool CellsFromTuple(
+    const Ydb::Type* tupleType, // nullptr if no type verification needed
+    const Ydb::Value& tupleValue,
+    const TConstArrayRef<TConversionTypeInfo>& types,
+    bool allowCastFromString,
```

Comment:
is it about implicit cast from string literal in postgres?

Reply:
Yes, but no. It's about implicit casts for native YDB types only. It's from original impl of CellsFromTuple for NKikimrMiniKQL::TValue: https://github.com/ydb-platform/ydb/blob/main/ydb/core/engine/mkql_proto.cpp#L17